cves/kernel/CVE-2018-20854.yml

```diff
@@ -234,11 +239,7 @@ interesting_commits:
       * A significant rewrite in the git history
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
-  commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  commits: []
```

Comment:
missing interesting commits

Reply:
I did not find any interesting commits as there were only 2 commits in the log between the VCC and the fix, but I've mentioned that as a comment now.